Dockerfile

```
@@ -0,0 +1,26 @@
+FROM python:3.8.5-slim-buster
+
+RUN apt update && apt install -y \
+    unzip \
+    git \
+    cloc \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
```

```suggestion
RUN $HOME/.poetry/bin/poetry config virtualenvs.create false
```
So that the dependencies are installed globally.